simplelink/source/ti/devices/cc13x2_cc26x2/CMakeLists.txt

```diff
@@ -7,12 +7,16 @@ zephyr_include_directories(
 zephyr_library()
 zephyr_library_compile_definitions(${COMPILER})
 zephyr_library_sources(
+  # Required for IEEE 802.15.4 support
```

Comment:
should these only be built if the 802154 driver kconfig is enabled?

Reply:
Good idea, thanks. This would help reduce the build time.